apps/src/templates/teacherNavigation/ResourceRow.tsx

```diff
@@ -25,19 +26,30 @@ const ResourceRow: React.FC<ResourceRowProps> = ({
   unitNumber,
   resource,
 }) => {
-  const resourcePositionLabel = (
-    <strong>{unitNumber + '.' + lessonNumber + ' '}</strong>
-  );
+  const resourceDisplayText = () => {
+    if (!resource.type) {
+      return resource.name;
+    } else if (resource.type === 'Standards') {
+      return i18n.unitStandards({unitNumber: unitNumber});
+    } else if (resource.type === 'Vocabulary') {
+      return i18n.unitVocabulary({unitNumber: unitNumber});
```

Comment:
Added [this](https://codedotorg.atlassian.net/browse/TEACH-1378) ticket to cover this work.  This will also impact how we display other resources (currently those are also using "Unit Numbers" to indicate which lesson the resource belongs to.